nss-day-cohort-48/fun-website-1-lemonade-stand
# Added the rest of the .css files

I added an intro, description, link, and syntax css files that should all appear in the styles folder.

##STEPS TO TEST:

1. Make sure there are no css files outside of the styles folder
2. Make sure inside of the styles folder there are 5 css files
3. Make sure all the css files are named correctly
